std::is space not found by MSVC

In binary.cpp lines 68 MSVC complains:
```
error C2039: 'isspace': is not a member of 'std'
```
This is of course correct, since `std::isspace` is defined in \<cctype\> not in \<ctype.h\>

--- a/src/binary.cpp
+++ b/src/binary.cpp
@@ -1,6 +1,6 @@
 #include "yaml-cpp/binary.h"
 
-#include <ctype.h>
+#include <cctype>
 
 namespace YAML {
 static const char encoding[] =

